Makefile

```diff
@@ -32,7 +32,7 @@ lint:
 	black --check rasa_sdk tests
 
 test: clean
-	py.test tests --cov rasa_sdk
+	py.test tests --cov rasa_sdk -W ignore
```

Comment:
why this change?

Reply:
when we've changed `logger.warning` to `warnings.warn` it has started to print warnings about invalid number of sanic workers or rasa version while the tests were running and because of that it never fully passed

Reply:
@imLew let's not forget about sdk eventually when fixing the warnings in tests

Reply:
thanks for the reminder ;)